.circleci/config.yml

```diff
@@ -101,7 +101,15 @@ jobs:
     - &report_code_coverage
       run:
         name: Report code coverage
-        command: bash <(curl -s https://codecov.io/bash) -F "$(basename $PWD | sed s/[^a-z]/_/g)"
+        command: |
+          curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --no-default-keyring --keyring trustedkeys.gpg --import
+          curl -Os https://uploader.codecov.io/latest/linux/codecov
+          curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM
+          curl -Os https://uploader.codecov.io/latest/linux/codecov.SHA256SUM.sig
+          gpgv codecov.SHA256SUM.sig codecov.SHA256SUM
+          shasum -a 256 -c codecov.SHA256SUM
+          chmod +x codecov
+          ./codecov -F "$(basename $PWD | sed s/[^a-z]/_/g)"
```

Comment:
bash uploader will fully stop working Feb 22nd, but they will be conducting unscheduled brown-outs until then, per: https://docs.codecov.com/docs/about-the-codecov-bash-uploader

new uploader docs: https://docs.codecov.com/docs/codecov-uploader